openshift-install-powervs

```diff
@@ -1281,36 +1283,36 @@ function process_varfile {
   done
 }
 
-function display_version {
-  echo "Version $VERSION"
-  exit 0
-}
-
-function main {
-  mkdir -p ./logs
-  vars=""
-
-  # Only use sudo if not running as root
-  [ "$(id -u)" -ne 0 ] && SUDO=sudo || SUDO=""
-
+#-------------------------------------------------------------------------
+# Perform platform checks
+#-------------------------------------------------------------------------
+function platform_checks {
   if [[ $(uname -m) != "x86_64" ]]; then
     warn "Only 64-bit x86 machines are supported"
     error "Unsupported machine: $(uname -m)" 1
   fi
   PLATFORM=$(uname)
   case "$PLATFORM" in
     "Darwin")
+      OS_VERSION=$(sw_vers -productVersion 2>/dev/null || echo "")
+      if [[ $IGNORE_OS -eq 0 ]] && [[ "$OS_VERSION" != "10.15"* && "$OS_VERSION" != "11."* ]]; then
+        error "Unsupported os version-$OS_VERSION of Mac, please use '-ignore-os-checks' option if you still want to run it."
```

Comment:
Only this is enough - "Unsupported OS version ($OS_VERSION) for Mac"

Reply:
Done